test/test_interactiverun.py

```diff
@@ -59,6 +62,22 @@ def fn(a, b, c, d):
                                   "ret_val_of_rank_1",
                                   None], res2)
 
+    @pytest.mark.skipif(sys.version_info < (3, 0), reason='Elastic Horovod supported on Python3')
```

Comment:
Shouldn't be needed anymore, since we dropped Python 2 support.

Reply:
good catch, removed all python2 references, also one from `horovod.run.http.http_client.py`, though out of scope for this PR but while I am at it ...